db/samples/stock.rb

```diff
@@ -14,6 +14,6 @@
 
 Spree::Variant.all.each do |variant|
   variant.stock_items.each do |stock_item|
-    Spree::StockMovement.create(:quantity => 10, :stock_item => stock_item)
+    Spree::StockMovement.create(:quantity => 1, :stock_item => stock_item)
```

Comment:
__[rubocop]__
* Use the new Ruby 1.9 hash syntax.
* Use the new Ruby 1.9 hash syntax.